viv-wallet-app/src/views/Wallets.vue

```diff
@@ -0,0 +1,140 @@
+                                    >{{ expertise.expertiseName }}</option
+                                >
+                            </select>
+                            <span class="select-focus"></span>
+                        </div>
+                    </div>
+                </div>
+                <consultant-list v-bind:consultants="consultants" v-bind:expertise="selectedExpertiseId">
+                    <template v-slot="{ consultantId }">
+                        <router-link
+                            v-bind:to="`/wallets/${selectedExpertiseId}/${consultantId}`"
+                            class="tertiary-button update-button"
+                            tag="a"
+                            >Voir le wallet</router-link
+                        >
+                    </template>
+                </consultant-list>
+            </section>
+        </loading>
+    </div>
+</template>
+
+<script lang="ts">
+import { Component, Prop, Vue, Watch } from "vue-property-decorator";
+import { Consultant } from "../models/consultant";
+import { Expertise } from "../models/expertise";
+import { ConsultantsService } from "@/services/consultants";
+import { ExpertisesService } from "@/services/expertises";
+import Loading from "../components/Loading.vue";
+
+import ConsultantList from "@/components/ConsultantList.vue";
+
+@Component({
+    name: "wallets",
+    components: { ConsultantList, Loading }
+})
+export default class Wallets extends Vue {
+    consultants: Consultant[] = [];
+    expertises: Expertise[] = [];
+    selectedExpertiseId = "";
+    loading = true;
+    errored = false;
+
+    expertisesService = new ExpertisesService();
+
+    async mounted() {
+        try {
+            this.expertises = await this.expertisesService.getExpertises();
+            if (this.$route.params.id) {
+                this.selectedExpertiseId = this.$route.params.id;
+                await this.updateConsultants();
+            } else {
+                if (this.expertises.length > 0) {
+                    this.$router.push(`/wallets/${this.expertises[0].id}`);
+                }
+            }
+        } catch (ex) {
+            this.errored = true;
+        } finally {
+            this.loading = false;
+        }
+    }
+
+    @Watch("$route")
+    async routeChanged() {
+        if (this.$route.params.id) {
+            this.selectedExpertiseId = this.$route.params.id;
+            await this.updateConsultants();
+        } else {
+            this.expertiseChanged();
+        }
+    }
+
+    expertiseChanged() {
+        this.$router.push(`/wallets/${this.selectedExpertiseId}`);
+    }
+
+    async updateConsultants() {
+        const consultantsService = new ConsultantsService(this.selectedExpertiseId);
+        this.consultants = await consultantsService.getConsultants();
+    }
+}
+</script>
+
+<style scoped lang="scss">
+@import "../styles/buttons.scss";
+@import "../styles/select.scss";
+
+.wallets {
```

Comment:
i don't know, but i feel we are repeating these things everywhere...

Reply:
Perhaps there is a better way. I don't know yet. If you have an idea, feel free to propose :)

Reply:
for me the standard old but working way would be defining a set of css classes and reusing them everywhere.
the h2 in all display forms should be the same, so we have a class called `displayListH2` in a css file and use it from (in practice, something with a better name).

what do you think?

Reply:
so we could import some scss files globally to avoid repeating ourselves + some other styles (like displayListH2 with a better name). We can do it in another PR.